gpbft/api.go

```diff
@@ -72,3 +72,10 @@ type Host interface {
 	// Logs a message at the "logic" level
 	Log(format string, args ...interface{})
 }
+
+type DecisionReceiver interface {
+	// Receives a finality decision from the instance, with signatures from a strong quorum
+	// of participants justifying it.
+	// The decision payload always has round = 0 and step = DECIDE.
```

Comment:
Is it always Round=0? If it is, then I need to update my mental model.

Reply:
Different nodes can terminate at different rounds but they all need to sign the same payload. Therefore, the payload at least always needs to be the same round (and the default round value seems most appropriate)